File: dataset/output/MISTRAL_codestral-latest/Task61_MISTRAL_codestral-latest.cpp
#include <iostream>
#include <libxml/parser.h>
#include <libxml/tree.h>

int main() {
    const char *xmlString = "<root><child>Data</child></root>";
    xmlDocPtr doc = xmlReadMemory(xmlString, strlen(xmlString), NULL, NULL, 0);
    if (doc == NULL) {
        fprintf(stderr, "Failed to parse document\n");
        return 1;
    }
    xmlNodePtr root_element = xmlDocGetRootElement(doc);
    std::cout << "Root element: " << root_element->name << std::endl;
    xmlFreeDoc(doc);
    xmlCleanupParser();
    return 0;
}

File: dataset/output/MISTRAL_codestral-latest/Task61_MISTRAL_codestral-latest.java
import javax.xml.parsers.DocumentBuilderFactory;
import org.w3c.dom.Document;
import org.w3c.dom.Element;
import java.io.StringReader;

public class Main {
    public static void main(String[] args) throws Exception {
        String xmlString = "<root><child>Data</child></root>";
        Document doc = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(new InputSource(new StringReader(xmlString)));
        Element rootElement = doc.getDocumentElement();
        System.out.println("Root element: " + rootElement.getNodeName());
    }
}